packages/vue-app/template/index.js

```diff
@@ -46,7 +46,7 @@ Vue.component(Nuxt.name, Nuxt)
 
 Object.defineProperty(Vue.prototype, '<%= globals.nuxt %>', {
   get() {
-    return this.$root.$options.<%= globals.nuxt %>
+    return this.$root.$options.<%= globals.nuxt %> || (process.client && typeof window !== 'undefined' && window.<%= globals.nuxt %>)
```

Comment:
I'm worried this will lead to inconsistent behaviour between client/server rendering.

Reply:
Indeed we have to double-check this. Check-related issue #8995 is due to usage of the second app by vuetify.

Reply:
I don't see any inconsistant behaviour between client/server. 

In the server it will be `this.$root.$options.<%= globals.nuxt %>` if it's defined. If it's not defined, it will still be undefined, because window is not defined server side. 

On the client it won't change anything if `$root` exists - only if it does not. Could you please explain, what the prolem is?

Reply:
It looks OK, but as @pi0 mentioned, let double check, a test for this would be test.

BTW, I think we can split this complex expression into if/else, sth like:

```js
const globalNuxt = this.$root.$options.<%= globals.nuxt %>

if(process.client && !globalNuxt && window) {
  return window.<%= globals.nuxt %>
}

return globalNuxt
```